stats/lib/platform-routes.js

```diff
@@ -32,6 +33,8 @@ export const handlePlatformRoutes = async (req, res, pgPools) => {
     await respond(pgPools.evaluate, fetchDailyStationAcceptedMeasurementCount)
   } else if (req.method === 'GET' && url === '/transfers/daily') {
     await respond(pgPools.stats, fetchDailyRewardTransfers)
+  } else if (req.method === 'GET' && url === '/participants/top-earnings') {
```

Comment:
```suggestion
  } else if (req.method === 'GET' && url === '/participants/top-earning') {
```

This route is about the top earning participants, and not the top earnings among participants (subtle but I think important)

Reply:
please add this route to the README as well

Reply:
Sure, addressed in https://github.com/filecoin-station/spark-stats/pull/170/commits/633f1946b648dfc8e1480d8e426f188853f7068c